### DISCUSSION TITLE
xgboost example

### DISCUSSION THREAD

**Discussion by EvanGertis** [Trusted-AI/adversarial-robustness-toolbox#1717] [Q&A]
**AttributeError: partially initialized module 'xgboost' has no attribute 'DMatrix' (most likely due to a circular import)**

1. clone the ART repo.
2. cd into the examples dir.
3. run the xgboost example

**Expected behavior**
The program should print out the accuracy for the classification of the MNIST Model.

**Actual**
Traceback (most recent call last):
  File "/Users/evangertis/development/PythonAutomation/ART_examples/xgboost.py", line 7, in <module>
    import xgboost as xgb
  File "/Users/evangertis/development/PythonAutomation/ART_examples/xgboost.py", line 31, in <module>
    dtrain = xgb.DMatrix(x_train, label=np.argmax(y_train, axis=1))
AttributeError: partially initialized module 'xgboost' has no attribute 'DMatrix' (most likely due to a circular import)

**Screenshots**
<img width="818" alt="Screen Shot 2022-05-25 at 8 13 27 PM" src="https://user-images.githubusercontent.com/39931491/170389636-a2f83954-8ed6-48ab-ac22-b1634628701d.png">

**System information (please complete the following information):**
 - MacOS
 - Python 3.9
 - Latest commit 3ff566f 
 - XGBoost: 1.5.2

**Comment by beat-buesser** [comment]
Hi @EvanGertis Thank you for using ART! It looks like you are using a very old version of ART. Would it be possible to upgrade ART to 1.10.2?
